README.md

```diff
@@ -19,7 +19,7 @@
 
 ## <a name='Legal'></a>Legal
 
-Dual-licensed under `MIT` or the [UNLICENSE](http://unlicense.org/).
+Dual-licensed under `MIT` or the [Unlicense](http://unlicense.org/).
```

Comment:
Here probably then also wiki?

Funny the reason why it is not available: `DNS points to prohibited IP | unlicense.org | Cloudflare`. Can probably guess from one try, where it is located.

Reply:
Maybe better than wiki:

- https://opensource.org/license/mit/
- https://opensource.org/license/unlicense/

Or:

- https://spdx.org/licenses/

Need to do better research...

Reply:
sure thing - feel free to pick and modify - i enabled edits by maintainers :)